### ISSUE TITLE
Regarding model-zoo ReID loss layer

### ISSUE THREAD

**Issue by Maharshi-Aupera** [Xilinx/Vitis-AI#423]
Hi Xilinx Community,

Thanks for the great work on Vitis-AI. I am trying to find the loss layer for re-training the ReID model from Model-Zoo but could not find the loss layer in any of the float prototxt files or readme markdown.

Did I miss any place looking at or the loss layer is not exposed to Vitis-AI user from Xilinx yet?

Thanks.

**Comment by Kewenjing1020** [comment]
Hi @Maharshi-Aupera , 
the reid_caffe model is trained on pytorch and then converted to caffe model. So we don't provide the reid_caffe training code.

**Comment by Maharshi-Aupera** [comment]
Thank you @Kewenjing1020.
